design-principles.md

```diff
@@ -41,8 +99,21 @@ and verify the test passes.
 
 When you implement a new feature, write tests for it.
 
-Run the tests before checking code in. (Travis does this for you, so
-wait for Travis to give the green light before merging a PR.)
+
+Documentation
+-------------
+
+If you have a question about our systems, then when you find the
+answer, put it into the documentation in the same place where you
```

Comment:
This is aimed at team members, not our users. I'll rephrase it to make that clear.